hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -3481,6 +3512,9 @@ public ScanResponse scan(final RpcController controller, final ScanRequest reque
       }
       throw new ServiceException(e);
     }
+    if(rsh.isFullRegionScan) {
+      rpcFullScanRequestCount.increment();
```

Comment:
Ya this looks good. So this is happening only for scan na ?

Reply:
Ya this is in RsRpcServices under scan() so should be ok. One thing I noticed is that - once  a scan is created and a next() call happens does the rpcScanRequestCount becomes 2 or is it 1?

Reply:
this is called in the scan method, same place as where we increment rpcScanRequestCount, so yes it should happen only for scan. @anoopsjohn to verify.

Reply:
fixed